### ISSUE TITLE
Update addon for Godot 3.2, twisted jigglebones.

### ISSUE THREAD

**Issue by Corruptinator** [Bauxitedev/godot-jigglebones#5]
_**set_bone_global_pose**_ no longer exists as of Godot 3.2. You'll need to update the jigglebone.gd script with the following at line 115 to replicate the old effect:

`skeleton.set_bone_global_pose_override(bone_id, bone_new_transf_obj, 1, true)`

Also, not sure if this is something that can be improved but testing the animation player caused the bone to be oversensitive:

(**Update**: Is it possible to write a code that resets the bone positions instead of using the move tool?)

![image](https://user-images.githubusercontent.com/5027750/73822073-28a2ed00-47aa-11ea-95ee-5c2618276bef.png)

**Comment by Hasenn** [comment]
I forked it and did this

**Comment by vipsbpig** [comment]
After this code in 3.2. It twisted the bone too much.How to impove this ?

**Comment by Hasenn** [comment]
could you open an issue on my fork and detail how to reproduce / what is wrong exactly?

**Comment by vipsbpig** [comment]
@Hasenn I can't open an issue on your fork page,maybe you forget to open the issues function.Just like the original post.Before it run well in game.Maybe crazy effect on the editor side.Now every play the monkey will not set to it's original form.

After the animation form original scene.The monkey will be twisted.Here is the pic.
![bug](https://user-images.githubusercontent.com/1953549/76071180-cb7ca000-5fd0-11ea-88b4-3a33dfaab3be.png)
I adujst the camera a little to see the monkey twisted clearly.

**Comment by Hasenn** [comment]
I'll take a look when i can

**Comment by Bauxitedev** [comment]
Yep, the twisting thing is a known problem, I'm not exactly sure why it happens. I'll have to port the thing to Godot 3.2 as well but currently I'm recovering from COVID so it won't be any time soon.

**Comment by Corruptinator** [comment]
OH WOW. I offer my prayers for your recovery! 
  _  _
  /)(\

**Comment by bengtsts** [comment]
Calling `skeleton.clear_bones_global_pose_override()` once and **only on the first jiggle bone processed in the scene tree** fixed this for me.

I figured that out after trying [this](https://www.youtube.com/watch?v=kaJxw4_vRdY) guide.

**Comment by TokisanGames** [comment]
The main issue in many of these tickets is that it runs in tool mode, which changes the data saved in your scene. That's why @bengtsts 's solution to clear it at runtime works. If you disable tool mode it will fix 99% of the problems as the initial data in your scene is set properly at runtime.

You can also try my fork which supports collision among other changes. See the readme.
https://github.com/tinmanjuggernaut/godot-jigglebones

@Bauxitedev I'm happy to contribute to your repo if you're still active on this project, or it can stay where it is.

**Comment by RockyMadio** [comment]
> The main issue in many of these tickets is that it runs in tool mode, which changes the data saved in your scene. That's why @bengtsts 's solution to clear it at runtime works. If you disable tool mode it will fix 99% of the problems as the initial data in your scene is set properly at runtime.
> 
> You can also try my fork which supports collision among other changes. See the readme. https://github.com/tinmanjuggernaut/godot-jigglebones
> 
> @Bauxitedev I'm happy to contribute to your repo if you're still active on this project, or it can stay where it is.

Thank you, is it going to work in Godot 3.5.3?
